src/lib.rs

```diff
@@ -684,6 +694,50 @@ pub trait Context {
     unsafe fn stencil_op(&self, stencil_fail: u32, depth_fail: u32, pass: u32);
 
     unsafe fn stencil_op_separate(&self, face: u32, stencil_fail: u32, depth_fail: u32, pass: u32);
+
+    unsafe fn debug_message_control(
+        &self,
+        source: u32,
+        msg_type: u32,
+        severity: u32,
+        ids: Option<&[u32]>,
```

Comment:
is this idiomatic in `glow` as opposed to just `&[u32]`?

Reply:
This is the same pattern used for `glow::tex_imaged_{2,3}d` and `glow::tex_sub_image_2d_u8_slice`. The reasoning there may be different from here, though, I'm not sure.

Reply:
I think we can just simplify to `&[u32]` – I simplified a few of them already when copying the signatures from web-sys but probably missed a few

Reply:
great!